autoarray/structures/grids/two_d/grid_2d_pixelization.py

```diff
@@ -431,6 +416,31 @@ def extent(self) -> np.ndarray:
             ]
         )
 
+
+class Grid2DVoronoi(AbstractGrid2DIrregular):
+    @cached_property
+    def pixel_neighbors(self) -> PixelNeighbors:
+        """

```

Comment:
Are the pixel neighbors computed via this method ever different to those computed via the Delaunay method below?